src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -110,6 +120,12 @@ class ActorWorkflowManager extends WorkflowManagerActor {
     } yield context.actorOf(WorkflowActor.buildWorkflowActorProps(id, binding, coercedInputs))
   }
 
+  override def notifyCompletion(obj: Option[AnyRef]): Unit = {
+    obj match {
+      case Some(o) => o.synchronized{o.notify()}
```

Comment:
Yah, I knew that this wasn't best practice when I wrote it.  I struggled with the Actor stuff and a bit and couldn't get it working, And I just needed something that worked for the moment.  Miguel said he had some idea for an alternative implementation using messages